src/language-css/printer-postcss.js

```diff
@@ -550,6 +550,21 @@ function genericPrint(path, options, print) {
           continue;
         }
 
+        // Ignore SCSS @forward wildcard suffix
+        if (
+          insideAtRuleNode(path, "forward") &&
+          iNode.type === "value-word" &&
+          iNode.value &&
+          iPrevNode.type === "value-word" &&
+          iPrevNode.value &&
```

Comment:
```suggestion
```
`iPrevNode.value && iPrevNode.value === "as"` is the same thing as just  `iPrevNode.value === "as"`.